project/MicroService.scala

```diff
@@ -27,8 +27,8 @@ trait MicroService {
     import scoverage.ScoverageKeys
     Seq(
       // Semicolon-separated list of regexs matching classes to exclude
-      ScoverageKeys.coverageExcludedPackages := "<empty>;model.*;config.*;.*(AuthService|BuildInfo|Routes).*",
-      ScoverageKeys.coverageMinimum := 20,
+      ScoverageKeys.coverageExcludedPackages := "<empty>;.*config.*;.*(AuthService|BuildInfo|Routes|JsErrorOps).*",
+      ScoverageKeys.coverageMinimum := 94,
```

Comment:
Let's keep it as is for now then and see how much we do/don't struggle to keep it at that level :D